docs/source/examples/substrafl/go_further/run_mnist_cyclic.ipynb

```diff
@@ -254,16 +254,16 @@
         "import numpy as np\n",
```

Comment:
<p>rename <code>perform_predict</code> to <code>perform evaluation</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/376/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>